WireDomain/Sources/WireDomain/Account/StoredAccount.swift

```diff
@@ -0,0 +1,88 @@
+//
+// Wire
+// Copyright (C) 2025 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+import WireDataModel
+
+struct StoredAccount: Codable {
+
+    var identifier: UUID
+    var name: String
+    var image: Data?
+    var team: String?
+    var teamImage: Data?
+    var loginCredentials: StoredLoginCredentials?
+    var unreadConversationCount: Int
+
+    init(_ account: Account) {
+        self.identifier = account.userIdentifier
+        self.name = account.userName
+        self.image = account.imageData
+        self.team = account.teamName
+        self.teamImage = account.teamImageData
+        self.loginCredentials = account.loginCredentials.map {
+            StoredLoginCredentials($0)
+        }
+        self.unreadConversationCount = account.unreadConversationCount
+    }
+
+}
+
+struct StoredLoginCredentials: Codable {
```

Comment:
comment: A somewhat misleading name. I hope we don't actually store login credentials.

Reply:
Agreed that the name is misleading, perhaps `StoredAuthenticationMetadata`?